desk/app/chat.hoon

```diff
@@ -358,6 +400,8 @@
     ::  don't proxy to self, creates an infinite loop
     ?:  =(p.action our.bowl)
       ~|("%dm-action poke failed: can't dm self" !!)
+    ?:  (~(has in blocked) src.bowl)
+      ~|("%dm-action poke failed: ship blocked" !!)
```

Comment:
need to add this check to `di-take-counter` which is where we normally receive dms from other parties. this route is more for ourselves.

Reply:
oh nvm I see you put it later